### PR TITLE
Subtitle support

### PR DESCRIPTION
**Pull request by Evidlo**

### CODE REVIEW

**Comment by Evidlo** [comment]
How about we use the built in fonts at `/usr/share/ttf/fonts/noto`?

**Comment by fringley** [comment]
Yeah, thats a good idea.  Knowing a little more about how this works now, I guess we could try loading from that location and then fall back to a known file in `./`, this would allow it to work locally and on the device.

Of course, we need to figure out #25 otherwise this is a bit moot!

**Comment by Evidlo** [comment]
Alternatively we could embed the font into the binary itself:

https://stackoverflow.com/questions/17796043/how-to-embed-files-into-golang-binaries

**Comment by fringley** [comment]
That's kinda cool.  I think just referencing the font works well enough.

**Comment by Evidlo** [comment]
This is looking good so far, but I think it would make more sense to change `-text` to something like `-subtitle` and have its argument be an xpath to the caption text rather than a fixed string.

File: sources.go [hunk 1]
```diff
@@ -101,33 +106,72 @@ func adjust(img image.Image, mode string, scale float64) image.Image {
...
+	}
+
+	width := d.MeasureString(label)
+	x := (fixed.I(1404) - width) / 2.0
+
+	d.Dot = fixed.Point26_6{
+		X: x,
+		Y: fixed.I(y),
+	}
+	d.DrawString(label)
```
Comment (Evidlo) [inline sources.go:174]: Use the returned string from `get_xpath(url, subtitle, "html)` as the subtitle text instead of the user provided string directly.

Comment (Evidlo) [reply sources.go:174]: Bump

**Comment by Evidlo** [comment]
Superceded by #37

**Pull request closed without merging**